extensions/github-review-requests/src/actionablePullRequests.tsx

```diff
@@ -50,7 +50,7 @@ const actionablePullRequests = () => {
 export default actionablePullRequests;
 
 const getTitle = (updatedPulls: PullRequestShort[]) =>
-  updatedPulls.length === 0 ? `All good` : `${updatedPulls.length} PR${updatedPulls.length > 1 ? "s" : ""} to check`;
+  updatedPulls.length === 0 ? `All good` : `${updatedPulls.length} PR${updatedPulls.length > 1 ? "s" : ""}`;
```

Comment:
Maybe we don't need `All good` then? `N PRs` for N PRs, green Github icon otherwise

Reply:
Good call! I've removed it from the menubar.